src/__tests__/GlobalStats.test.js

```diff
@@ -0,0 +1,15 @@
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import GlobalStats from '../components/Globalstats';
+
+it('Test Globalstats renders correctly', () => {
+  const tree = renderer
+    .create(
+      <Provider store={store}>
+        <GlobalStats />
+      </Provider>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
```

Comment:
- [ ] Great job working on the tests so far but I think it would be better if you Create unit tests for pure functions in your Redux code as mentioned in the requirements so please add more tests to that and please remember :

- You may need to mock the access to the API, so that your tests don't send actual requests.
- You may need to mock the connection to the Redux Store.

![image](https://github.com/dazehinn/world-covid-19-data/assets/58553711/0848c2bf-ff17-4a36-92c2-7a7bf0c3899c)

Reply:
- [ ] As mentioned by the previous code reviewer, kindly write test for the pure functions in your Redux code, also write integration test as mentioned by the previous code reviewer. Kindly make sure the test you have written all pass, as we can see in the screenshot below, we have failing tests
![Screenshot from 2023-06-02 22-32-10](https://github.com/dazehinn/world-covid-19-data/assets/46388113/510dbffb-01ac-4727-b303-4dbe94dd879a)